cmd/wallet-sdk-js/jsinterop/agent.go

```diff
@@ -0,0 +1,59 @@
+//go:build js && wasm
+
+/*
+Copyright Gen Digital Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+// Package jsinterop implements interops between wallet sdk and js.
+package jsinterop
+
+import (
+	"fmt"
+	"syscall/js"
+
+	"github.com/hyperledger/aries-framework-go/component/kmscrypto/kms"
+	"github.com/hyperledger/aries-framework-go/component/storage/indexeddb"
+
+	"github.com/trustbloc/wallet-sdk/cmd/wallet-sdk-js/walletsdk"
+)
+
+const (
+	dbNamespace = "wallet-sdk"
+)
+
+var agentInstance *walletsdk.Agent
+var agentMethodsRunner AsyncRunner
+
+func InitAgent(_ js.Value, args []js.Value) (any, error) {
```

Comment:
Eventually if we expose the functions independently without an "agent"-type of wrapper/container, we can rename this as needed. For now I think the "agent" name makes sense